### ISSUE TITLE
[apex] AvoidBooleanMethodParameters: False positive on getter and setter

### ISSUE THREAD

**Issue by Joper-jpg** [pmd/pmd#6261]
**Affects PMD Version:**  7.15.0

**Rule:** [AvoidBooleanMethodParameters](https://docs.pmd-code.org/latest/pmd_rules_apex_design.html#avoidbooleanmethodparameters)

**Description:**
Rule marks Boolean getter, setter as violation.

**Code Sample demonstrating the issue:**

```apex
public Boolean displayable { get; set; }
```

**Expected outcome:**

PMD reports a violation, getter and setter should not be considered as violation of this rule. That's a false positive.

**Running PMD through:** *CLI*
